src/module/classes/sorcerer.mjs

```
@@ -0,0 +1,79 @@
+        // Determine if a wild surge should occur
+        if (rollAbove || volen || blowout) {
+            let count = 0;
+            if (rollAbove) count++;
+            if (volen) count++;
+            if (blowout) count++;
+
+            // Calculate the table level based on the spell level and additional conditions
+            let tableLevel = level - 1 + count;
+            if (tableLevel > MAX_TABLE_LEVEL) tableLevel = MAX_TABLE_LEVEL;
+
+            let tableUUID = TABLE_UUIDS[tableLevel];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'tableUUID' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)